tensorflow_model_analysis/post_export_metrics/metric_keys.py

```diff
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Library defining metric keys for post export evaluation.
+"""Library defining metric keys for post export metrics evaluation.
```

Comment:
How about just "post export metrics" (no evaluation)? I think that would be clearer.

Reply:
Done